### ISSUE TITLE
Multivariate models in estimatr::lm_robust()

### ISSUE THREAD

**Issue by ngreifer** [easystats/insight#618]
`estimatr::lm_robust()` can fit multivariate models just like `lm()` can and produces the same coefficient matrix with a column for each outcome. However, `insight::get_parameters()` does not process these models correctly. See below:

``` r
data(mtcars)

#Using lm()
fit_lm <- lm(cbind(mpg, qsec) ~ cyl + disp, data = mtcars)
coef(fit_lm)
#>                     mpg        qsec
#> (Intercept) 34.66099474 22.71634662
#> cyl         -1.58727681 -1.07411344
#> disp        -0.02058363  0.00770833
insight::get_parameters(fit_lm)
#>     Parameter    Estimate Response
#> 1 (Intercept) 34.66099474      mpg
#> 2         cyl -1.58727681      mpg
#> 3        disp -0.02058363      mpg
#> 4 (Intercept) 22.71634662     qsec
#> 5         cyl -1.07411344     qsec
#> 6        disp  0.00770833     qsec

#Using lm_robust()
fit_lm_r <- estimatr::lm_robust(cbind(mpg, qsec) ~ cyl + disp, data = mtcars)
coef(fit_lm_r)
#>                     mpg        qsec
#> (Intercept) 34.66099474 22.71634662
#> cyl         -1.58727681 -1.07411344
#> disp        -0.02058363  0.00770833
insight::get_parameters(fit_lm_r)
#>   Parameter  Estimate.1  Estimate.2
#> 1         1 34.66099474 22.71634662
#> 2         2 -1.58727681 -1.07411344
#> 3         3 -0.02058363  0.00770833
#> 4         4 34.66099474 22.71634662
#> 5         5 -1.58727681 -1.07411344
#> 6         6 -0.02058363  0.00770833
```

<sup>Created on 2022-08-25 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Although the coefficient matrix is the same between the two modeling functions, the `get_parameters()` output is quite different. Multivariate `lm_robust` models do not have their own class, but they can be determined based on whether the coefficients are in a matrix.

**Comment by ngreifer** [comment]
I also noticed that `is_multivariate()` returns `FALSE` for this model.

**Comment by vincentarelbundock** [comment]
This should be fixed by PR #622 I used your example as test, but feel free to re-open or keep commenting if you run into related issues.
